vars/folioUI.groovy

```diff
@@ -1,11 +1,16 @@
 #!groovy
```

Comment:
are we sure about this lib? Seems like we're using this one: uiBuild.groovy...

Reply:
Yeap, folioUi it is refactored uiBuild. uiBuild will be removed soon